frontend/src/components/HuntDetails.tsx

```diff
@@ -634,7 +636,28 @@ export function HuntDetails() {
                     </span>
                   </div>
                     <div className="border-t-2 border-gray-200 pt-4">
-                      <h3 className="text-lg font-semibold mb-4">Your Team</h3>
+                      <div className="flex items-center justify-between mb-3">
+                        <h3 className="text-lg font-semibold truncate">
+                          {`Team ${teamData?.name || ''}`.trim() || 'Team'}
+                        </h3>
+                        {Number(teamData?.memberCount || 0) < Number(teamData?.maxMembers || 0) && teamData?.owner === userWallet && (
+                          <Button 
+                            size="sm"
+                            onClick={async () => {
+                              try {
+                                const teamIdStr = teamData?.teamId?.toString?.();
+                                if (teamIdStr) {
+                                  await generateInviteAfterTeamCreation(teamIdStr);
+                                }
+                              } catch (e) {
+                                toast.error('Failed to generate invite');
+                              }
+                            }}
+                          >
+                            Invite more
+                          </Button>
+                        )}
```

Comment:
Remove this.

Not having an Invite More button is meant to be. 

We explicitly ask the user to save the screenshot of the QR as invites expire after a certain interval.